resolver/serviceResolver.go

```diff
@@ -485,6 +494,23 @@ func (resolver *serviceResolver) validateTxRequestReturningGuardian(
 	return resolver.verifyCodesReturningGuardian(userAddress, txs[0].GuardianAddr, userIp, code, secondCode)
 }
 
+func (resolver *serviceResolver) verifyUserReturningSecurityStatus(userAddr string) (core.Status, error) {
```

Comment:
```suggestion
func (resolver *serviceResolver) getUserStatus(userAddr string) (core.Status, error) {
```
no verification done here

Reply:
renamed it